Update submitting.md (#1554)
openjournals/joss: docs/submitting.md

**2. Demonstrated research impact**

> There **must be evidence that the software is being used for research** — at minimum by the developers themselves, and ideally by others. Acceptable signals include: references in published papers or preprints, DOIs linking to the software, documented adoption by other research groups, or clear integration into research workflows. 

Reviewer: ```suggestion
There **must be evidence that the software is being used for research** — at minimum by the developers themselves, and ideally by others. Acceptable signals include: references in published papers or preprints, DOIs linking to the software, documented adoption by other research groups, or clear integration into research workflows. Adoption in currently proprietary workflows (e.g. before an experiment's first papers and/or data releases) is acceptable but must be demonstrated to the editorial team.
```

Author: Can we find another word than "proprietary"? This sounds too commercial to me.  Maybe we could say "Use in active research projects (e.g. before an experiment's first papers and/or data releases) may be acceptable but must be demonstrated to the editorial team."

Reviewer: Happy to find another word, I used 'proprietary' because it's the standard word in astronomy for a time when a group has exclusive access to data, usually from an external observatory through a fixed-term 'proprietary period'.

Reviewer: "Un-seen adoption in workflows during an exclusive use period (e.g. before an experiment's first papers and/or data releases) is acceptable but must be demonstrated to the editorial team."?

Reviewer 2: Would "closed-source" be sufficient?
Though I guess that is still thinking in terms of software end dependencies...

Just "closed workflows"?

Author: The software has to be developed in the open in any case, so I'm not sure saying "closed" is useful.

Reviewer 2: The software under submission to JOSS, yes, but the application of it in another place could be closed, no?
E.g. I know some groups who have their core software (which would be submitted to JOSS) as open source, but then the analysis on a particular dataset/simulation, and the work towards a specific publication facilitated by that software takes place in a private repository.

This private repository where the software is used to generate results was what I understood as "workflow" in this context?

I also think @JBorrow's "unseen" would work.

> There **must be evidence that the software is being used for research** — at minimum by the developers themselves, and ideally by others. Acceptable signals include: references in published papers or preprints, DOIs linking to the software, documented adoption by other research groups, or clear integration into research workflows. 

Reviewer 3: > DOIs linking to the software

Anyone can just make a DOI to link to their software, like we have people do from Zenodo, so this doesn't seem like a good signal. 

Also do presentations count? I have seen people note those. 

I've also had people vaguely say a research project is using it currently but without anything specific to back that up, in which case I haven't thought it really "counts".

Reviewer 2: I agree with this point and was just writing a similar comment. I think we need to be clearer what does and does not count, even if that is internally, so it can be applied consistently across tracks and reviews.

E.g. for the anecdote about the "Big Collider" above what evidence that they are using it in research to prepare a publication would be acceptable, other than being a big well-known project and we trust their word? I think criteria should be something that can be applied consistently to any piece of software no matter the size/prestige of the submitting author/group.

Author: I removed the DOIs part.

> There **must be evidence that the software is being used for research** — at minimum by the developers themselves, and ideally by others. Acceptable signals include: references in published papers or preprints, DOIs linking to the software, documented adoption by other research groups, or clear integration into research workflows. Adoption in currently proprietary workflows (e.g. before an experiment's first papers and/or data releases) is acceptable but must be demonstrated to the editorial team.

Author: ```suggestion
There **must be evidence that the software is being used for research** — at minimum by the developers themselves, and ideally by others. Acceptable signals include: references in published papers or preprints, DOIs linking to the software, documented adoption by other research groups, or clear integration into research workflows. Adoption in currently private workflows (e.g., not shared publicly before an experiment's first papers and/or data releases) is acceptable but must be demonstrated to the editorial team.
```

> There **must be evidence that the software is being used for research** — at minimum by the developers themselves, and ideally by others. Acceptable signals include: references in published papers or preprints, DOIs linking to the software, documented adoption by other research groups, or clear integration into research workflows. Adoption in currently private workflows (e.g., not shared publicly before an experiment's first papers and/or data releases) is acceptable but must be demonstrated to the editorial team.

Author: ```suggestion
There **must be evidence that the software is being used for research** — at minimum by the developers themselves, and ideally by others. Acceptable signals include: references in published papers or preprints, documented adoption by other research groups, or clear integration into research workflows. Adoption in currently private workflows (e.g., not shared publicly before an experiment's first papers and/or data releases) is acceptable but must be demonstrated to the editorial team.
```

> Aspirational statements about future use are not sufficient; JOSS will not publish papers that are meant to advertise software that is not yet being used in research.

Reviewer 4: ```suggestion
Aspirational statements about future use are not sufficient; JOSS will not publish papers on software that is not yet being used in research.
```
I think this is less wordy, but still conveys the intended point

Author: I would like to keep this, as authors whose software hasn't yet been used in research often say that the reason they submitted is to advertise so that their software will be used in research
